Leo/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public CharacterController characterController;
+    public float playerSpeed = 8.0f;
```

Comment:
i would call it "speed" instead - it is inside PlayerController so it is obvious it is player speed.